pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -1390,3 +1398,11 @@ func (c *Cluster) getOSDServicePorts() []v1.ServicePort {
 
 	return ports
 }
+
+func osdStartScript(cmd, args []string) []string {
+	return []string{
+		"/bin/bash",
+		"-c",
+		fmt.Sprintf(cephOSDStart, strings.Join(cmd, " "), strings.Join(args, " ")),
```

Comment:
```suggestion
	return []string{
		"/bin/bash",
		"-c",
		fmt.Sprintf(cephOSDStart, strings.Join(cmd, " "), strings.Join(args, " ")),
```

also I think we can add all this flag
```
"/bin/bash",
			"-x", // Print commands and their arguments as they are executed
			"-e", // Exit immediately if a command exits with a non-zero status.
			"-m", // Terminal job control, allows job to be terminated by SIGTERM
			"-c", // Command to run```

Reply:
do we really need `-e`?